components/FormCreate.js

```diff
@@ -124,7 +137,6 @@ export default function FormCreate({ addActivity }) {
             type="url"
             defaultValue="https://images.unsplash.com/photo-1526772662000-3f88f10405ff?q=80&w=1548&auto=format&fit=crop&ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D"
```

Comment:
You might want to use criteria for the uploaded pictures as well, e.g. concerning the size.